packages/asc-ui/src/components/Link/Link.tsx

```diff
@@ -1,11 +1,19 @@
-import { styledComponents } from '@datapunt/asc-core'
 import React from 'react'
-import LinkStyle from './LinkStyle'
+import { styledComponents } from '@datapunt/asc-core'
+import { ChevronRight } from '@datapunt/asc-assets'
+import LinkStyle, { Props as LinkStyleProps } from './LinkStyle'
+import Icon from '../Icon'
+
+export type Props = styledComponents.StyledProps<any> & LinkStyleProps
 
-export type Props = styledComponents.StyledProps<any>
+const Link: React.FC<Props> = ({ children, linkType, ...otherProps }) => (
+  <LinkStyle $as="a" {...otherProps} linkType={linkType}>
```

Comment:
linkType is already in ...otherProps

Reply:
I know but is it used here spoecific, cant' be removed